css/header.css

```diff
@@ -0,0 +1,78 @@
+.overlay-header {
+  height: 100%;
+  width: 0;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  right: 0;
+  background-color: #3c3a39;
+  overflow-x: hidden;
+  transition: 0.5s;
+}
+
+.nav-hr {
+  color: #6f6c6b;
+  width: 100%;
+  height: 1px;
+}
+
+.overlay-content {
+  position: relative;
+  top: 10%;
+  width: 100%;
+  margin-top: 30px;
+  padding: 0 20px;
+}
+
+.overlay-header a {
+  padding: 8px;
+  text-decoration: none;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff5e1;
+  display: block;
+  transition: 0.3s;
+}
+
+.closebtn {
+  background-color: transparent;
+  border: none;
+  outline: none;
+}
+
+.closebtn > i {
+  color: #67798e;
+}
+
+.nav-icon {
+  font-size: 40px;
+  transform: rotate(-90deg);
+  margin-right: 8px;
+}
+
+/* .overlay-header .closebtn {
+  position: absolute;
+  top: 20px;
+  right: 45px;
+  font-size: 60px;
+}
+
+/* Start query */
```

Comment:
- [ ] Would be nice you take out all commented code, as is often considered a good practice. 👍